package/kedro_viz/api/apps.py

```diff
@@ -131,6 +131,24 @@ async def index():
 
     @app.get("/api/main", response_class=JSONResponse)
     async def main():
-        return json.loads(Path(filepath).read_text(encoding="utf8"))
+        return json.loads((Path(filepath) / "main").read_text(encoding="utf8"))
```

Comment:
Maybe we could update the variable name `filepath`? I understand it used to be 1 file and we use `--save-file`, this is now a folder and we can reflect this internally at least.

Reply:
renamed it to `api_dir`?